docs/src/components/MainSectionSubsection.js

```diff
@@ -35,16 +40,19 @@ const MainSectionSubsection = ({ children, description, title }: Props): Node =>
               accessibilityLabel={`${title} - Anchor tag`}
               size="xs"
               href={`#${slugifiedId}`}
+              onClick={() => {
+                copyToClipboard(slugifiedId);
+              }}
               role="link"
             />
           </Flex>
         </Box>
       )}
-      {description && (
-        <Box width="80%" marginTop={-3}>
-          <Markdown text={description} />
-        </Box>
-      )}
+
+      <Box width="80%" marginTop={-2} marginBottom={6}>
```

Comment:
Do we want to render this empty Box if there is no description? I would think "no" in general

Reply:
We actually do in this case to account for a funky spacing thing

Reply:
That doesn't sound like the right solution. Can that be solved by using flex properties on the parent/siblings?

Reply:
Wrapped it in a Box instead since Flex doesn't have margin